presto-docs/src/main/sphinx/release/release-0.285.rst

```diff
@@ -0,0 +1,75 @@
+* Improve :doc:`REST API</rest>` by providing links.
+* Improve :doc:`Resource Groups </admin/resource-groups>`.
+* Add Presto to MySQL type mapping in :doc:`MySQL Connector</connector/mysql>`.
+* Add :func:`array_least_frequent` function.
+* Add support for HANA connector. :pr:`21034`
+* Add task killer which is triggered when a worker is running out of memory and the garbage collector cannot reclaim sufficient memory. Two strategies are provided: full garbage collection, and frequent full garbage collection. :pr:`21254`
+* Add a session property ``history_input_table_statistics_matching_threshold`` to set the threshold for input statistics match in HBO. :pr:`21063`
+* Add a session property ``remove_redundant_cast_to_varchar_in_join`` (enabled by default) to remove redundant `cast to varchar` expressions in a join condition. :pr:`21050`
+* Add a session property ``enable_hbo_for_scaled_writer`` (disabled by default) to use HBO for scaled writers.
+* Add a session property ``use_partial_aggregation_history`` if partial aggregation statistics must be used to split aggregates into partial and final. :pr:`21160`
+* Add a config property ``task.memory-based-slowdown-threshold`` that provides a heap memory threshold below which new splits are not processed. :pr:`20946`
+* Add tracking of null probe keys in join and use it to enable outer join null salt in history-based optimization. :pr:`21203`
+* Add optimization where values node followed by an always false filter is converted to an empty values node.
+* Add information about cost-based optimizers and the source of stats they use (CBO/HBO) in explain plans when session property ``verbose_optimizer_info_enabled=true``. :pr:`20990`
+* Add a boolean to the PlanOptimizer's return value to indicate if an optimization has been triggered.
+* Rename config ``experiemental.task.high-memory-task-killer-strategy`` to ``experimental.task.high-memory-task-killer-strategy``.
+* Upgrade Joda-Time to 2.12.5 to use 2023c tzdata. The JVM should also be updated to ensure the timezone data is consistent. :pr:`21329`
+* Upgrade Alluxio to 304 from 2.8.1.
+* Upgrade AWS SDK to 1.12.560 from 1.12.261.
+* Upgrade Avro version to 1.11.3 from 1.9.2.
+
+Prestissimo (Native Execution) Changes
+______________________________________
+* Fix task cleanup to use task termination time instead of task end time.
+* Add support for JWT authentication. :pr:`20290`
+* Add session property ``native_debug.validate_output_from_operators`` to identify malformed output from operators. :pr:`21036`
+
+Security Changes
+________________
+* Fix critical vulnerability in Babel and related npm packages by updating to newer versions. :pr:`21322`
+
+Hive Connector Changes
+______________________
+* Improve support for schema evolution of partition column types for Parquet file format. :pr:`19983`
+* Add support for Parquet writer versions V1 and V2. See :doc:`Hive Connector</connector/hive>`.
```

Comment:
Thought about suggesting changing this link to 
`Hive Connector <../connector/hive.html#parquet-writer-version>`_  to link directly to the heading on the page, but  because the top of the Hive Connector doc page has "Parquet Writer Version" in the table of contents list of headings, I don't think it really adds value: the link you found and added is great. 

No change suggested, just unpacking my thought process.